docs/pages/admin-guides/access-controls/access-requests/automatic-reviews.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Configure Automatic Reviews
+description: Describes how to configure Access Monitoring Rules for Automatic Reviews.
+---
+
+Teleport supports automatic reviews of role Access Requests. This
+feature enables teams to enforce a zero standing privilege policy, while
+still allowing users to receive temporary access without manual approval.
+
+Support for automatic reviews of resource Access Requests is currently in
+development.
```

Comment:
I would just remove this.
```suggestion
```

Reply:
I think it is worth mentioning up front that automatic reviews with resource requests are not supported yet. It was pointed out by Zac, and some other while testing this feature.

Reply:
Let's make it an "info" admonition then at least? Otherwise it's just lost among the other text and people will skim over it.